packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -235,6 +236,7 @@ module.exports = {
                   importLoaders: 1,
                   modules: !optInCssModules,
                   localIdentName: '[local]-[hash:base64:5]',
+                  getLocalIdent: getLocalIdent,
```

Comment:
Can we not use es6 syntax here?

Reply:
Im not sure - i'd rather not because the code in `backpack-react-scripts` isn't transpiled and technically it can run on older versions of node. I'd rather just leave as is

Reply:
Ok cool, I thought it might be something like that